CI failure in integration tests due to:

```
2017-02-24 19:25:05 An exception occurred during task execution. The full traceback is:
2017-02-24 19:25:05 Traceback (most recent call last):
2017-02-24 19:25:05   File "/private/var/root/ansible/lib/ansible/executor/task_executor.py", line 126, in run
2017-02-24 19:25:05     res = self._execute()
2017-02-24 19:25:05   File "/private/var/root/ansible/lib/ansible/executor/task_executor.py", line 512, in _execute
2017-02-24 19:25:05     result = self._handler.run(task_vars=variables)
2017-02-24 19:25:05   File "/private/var/root/ansible/lib/ansible/plugins/action/copy.py", line 295, in run
2017-02-24 19:25:05     module_return['checksum'] = local_checksum
2017-02-24 19:25:05 UnboundLocalError: local variable 'local_checksum' referenced before assignment
2017-02-24 19:25:05 
2017-02-24 19:25:05 fatal: [testhost]: FAILED! => {
2017-02-24 19:25:05     "failed": true, 
2017-02-24 19:25:05     "msg": "Unexpected failure during module execution.", 
2017-02-24 19:25:05     "stdout": ""
2017-02-24 19:25:05 }
```

--- a/lib/ansible/plugins/action/copy.py
+++ b/lib/ansible/plugins/action/copy.py
@@ -158,8 +158,7 @@ class ActionModule(ActionBase):
             # If the local file does not exist, get_real_file() raises AnsibleFileNotFound
             try:
                 source_full = self._loader.get_real_file(source_full)
-            except AnsibleFileNotFound:
-                e = get_exception()
+            except AnsibleFileNotFound as e:
                 result['failed'] = True
                 result['msg'] = "could not find src=%s, %s" % (source_full, e)
                 self._remove_tmp_path(tmp)

